spec/ddoc.dd

```diff
@@ -199,11 +199,17 @@ $(H3 $(LNAME2 sections, Sections))
 
 $(P
 The document comment is a series of $(I Section)s.
-A $(I Section) is a name that is the first non-blank character on
+A $(I Section name) is a name that is the first non-blank character on
 a line immediately followed by a ':'. This name forms the section name.
 The section name is not case sensitive.
```

Comment:
This seems to not actually be true. I just tried writing `section_name:` vs `Section_name:`, and only the upper-case version is treated as a section name by ddoc in DMD 2.093.0.

Reply:
ddoc, while a fantastic idea, leaves a lot to be desired. At least we are moving toward markdown.